app/user.go

```diff
@@ -4,13 +4,16 @@ import "github.com/ubclaunchpad/cumulus/blockchain"
 
 // User holds basic user information.
 type User struct {
-	// Wallet holds the users wallets (currently just support 1).
-	Wallet blockchain.Wallet
-	// UserBlockSize is the maximum size of a block in bytes when marshaled
-	// as specified by the user.
+	HotWallet
```

Comment:
Is the plan for this to be embedded in the app struct? The miner at least will need to have access to this to sign mined blocks